src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/rightcurly/InputRightCurlyAloneBlocksWithSemi.java

```diff
@@ -57,4 +57,14 @@ public void testMethod11() {
     enum TestEnum4{ SOME_VALUE;
     }; //violation
 
+    interface Interface1
+    {
+        int i = 1;
+        public void meth1(); };
```

Comment:
please add one more case like https://gaurabdg.github.io/checkstyle-tester-reports/regression/rcurly/diff/Hbase/index.html#A1

Reply:
please do this.

Reply:
@gaurabdg , please keep replying to review items, it help a lot.